samples/subsys/display/lvgl/README.rst

```diff
@@ -24,7 +24,7 @@ for Arduino connectors, for example:
 
 or a simulated display environment in a native Posix application:
 
-- :ref:`native_posix`
+- :ref:`native_posix_64`
```

Comment:
Why the change from native_posix to native_posix_64?

Reply:
@jfischer-no  I have mistakenly replaced it here. It is now restored to its original.